web/src/TokenEditPage.js

```diff
@@ -35,14 +35,19 @@ class TokenEditPage extends React.Component {
 
   getToken() {
     TokenBackend.getToken("admin", this.state.tokenName)
-      .then((token) => {
-        if (token === null) {
+      .then((res) => {
+        if (res === null) {
```

Comment:
Will it be null?

Reply:
Already check, it will be null when no error happens, but not found a record in the database.